--- libutf8proc-1.3.1-3/Makefile~	2018-04-14 17:42:42.000000000 +0200
+++ libutf8proc-1.3.1-3/Makefile	2018-04-14 17:43:23.719751330 +0200
@@ -45,7 +45,7 @@
 endif
 
 # Extra installation rules
-I := /$(INCLUDEDIR)/libutf8proc
+I := /$(INCLUDEDIR)
 INSTALL_ITEMS := $(INSTALL_ITEMS) $(I):include/libutf8proc/utf8proc.h
 INSTALL_ITEMS := $(INSTALL_ITEMS) /$(LIBDIR)/pkgconfig:lib$(COMPONENT).pc.in
 INSTALL_ITEMS := $(INSTALL_ITEMS) /$(LIBDIR):$(OUTPUT)
